aion-etl/src/main/java/aion/dashboard/service/DBService.java

```diff
@@ -52,7 +52,7 @@ public BigDecimal getCirculatingSupply(Instant timestamp) throws SQLException {
                 }
             }
         }
-        return circulatingSupply.floorEntry(timestamp).getValue();
```

Comment:
Not sure I follow the implications, what would be the difference between selecting the closest value to the timestamp vs the first entry?

Reply:
The previous implementation of the Circulation supply had the value hardcoded by month, so the metric was calculated by the estimation of the current month. The new version only has one value in the DB that represents the current circulation supply by the current height - 500 blocks. Basically, no timestamp filter is needed because there is only one record.